tensorboard/data/server/BUILD

```diff
@@ -16,12 +16,16 @@ _proto_packages = [
 # and must match the output of `prost_build`.
 _genproto_files = ["genproto/%s.rs" % pkg for pkg in _proto_packages]
 
+_generated_file_descriptor_set = "genproto/descriptor.bin"
+
 # Source files with Rust protobuf bindings. These are automatically
 # materialized in the source tree from the generated bindings by the
 # `:update_protos` script, and kept in sync via `:update_protos_test`. These
 # don't match the raw generated code exactly: e.g., we add license headers.
 _checked_in_proto_files = ["%s.pb.rs" % pkg for pkg in _proto_packages]
 
+_checked_in_file_descriptor_set = "descriptor.bin"
```

Comment:
Amusingly, this binary file passes the license check because it embeds
the TensorFlow licenses that appear in comments in some of the proto
files. I thus haven’t changed `license_test.sh`, but can add an explicit
exemption for `*.bin` if we want.